# diff --git Source/WebKit/Platform/IPC/orig.ArgumentCoders.h Source/WebKit/Platform/IPC/ArgumentCoders.h
# index b861420..465d9c0 100644
# --- Source/WebKit/Platform/IPC/orig.ArgumentCoders.h
# +++ Source/WebKit/Platform/IPC/ArgumentCoders.h
# @@ -43,7 +43,7 @@
#  #include <wtf/Unexpected.h>
#  #include <wtf/WallTime.h>
#  
# -#if OS(DARWIN)
# +#if PLATFORM(COCOA)
#  #include "ArgumentCodersDarwin.h"
#  #endif
#  #if OS(WINDOWS)

diff --git Source/WebKit/Platform/IPC/orig.Connection.h Source/WebKit/Platform/IPC/Connection.h
index 80ec279..8368202 100644
--- Source/WebKit/Platform/IPC/orig.Connection.h
+++ Source/WebKit/Platform/IPC/Connection.h
@@ -52,8 +52,10 @@
 #if OS(DARWIN)
 #include <mach/mach_port.h>
 #include <wtf/OSObjectPtr.h>
+#if PLATFORM(COCOA)
 #include <wtf/spi/darwin/XPCSPI.h>
 #endif
+#endif
 
 #if USE(GLIB)
 #include <wtf/glib/GSocketMonitor.h>
@@ -259,7 +261,7 @@ public:
 #endif
     };
 
-#if OS(DARWIN)
+#if PLATFORM(COCOA)
     xpc_connection_t xpcConnection() const { return m_xpcConnection.get(); }
     std::optional<audit_token_t> getAuditToken();
     pid_t remoteProcessID() const;
